### ISSUE TITLE
not enough values to unpack in labels

### ISSUE THREAD

**Issue by oomensusan** [meituan/YOLOv6#505]
Hi Team,
I am trying to run yolov6 on custom dataset, but it has been mentioned that my labels are corrupted, 
![image](https://user-images.githubusercontent.com/8180920/192326539-837a5dbd-9ad4-4b3f-8d7e-b01be9261b25.png)
above is the error message, can someone tell where I am wrong please?

**Comment by shensheng272** [comment]
If dataset has been trained with yolov5/7 or older version of yolov6, the dataset cache might be incompatible for yolov6. Add` --check-images` and `--check-images` to create new cache.

**Comment by alumini** [comment]
Thank you , that seems to be working now :) , From which foler can I get the text file of the output, like the ones we get in yolov5

**Comment by Chilicyy** [comment]
You can get the output files under the directory of `runs/train/exp`

**Comment by alumini** [comment]
Apologies, one more query. I can find the box detected images in the run folder. I can see the txt files only. Can you help me with the location of the images as well, please?

**Comment by Chilicyy** [comment]
Did you run the script of `infer.py`?

**Comment by alumini** [comment]
yes, I ran the infer command, but under ./runs/inference/exp/ only text files are available, not the images
!python tools/infer.py --weights runs/train/exp/weights/best_ckpt.pt\
--conf 0.005\
--iou 0.45\
--source /kaggle/input/vinbigdata-512-image-dataset/vinbigdata/test/\
--save-img --save-txt

    !python tools/train.py --batch 5 --conf configs/config_best_weight.py --data data/vinbigdata.yaml --device 0 --epochs 5 --eval-interval 2 --check-images --workers 0

these were the commands I had given

**Comment by shensheng272** [comment]
> so wanted the graph results also

`--save-dir` could be used to set output dir. Detections result images are saved to `runs/inference/exp` in default.
